src/python/pants/notes/2.0.x.rst

```diff
@@ -5,6 +5,99 @@ This document describes releases leading up to the ``2.0.x`` ``stable`` series.
 
 See https://www.pantsbuild.org/v2.0/docs/release-notes-2-0 for an overview of the changes in this release.
 
+2.0.0a3 (9/11/2020)
+------------------
+
+API Changes
+~~~~~~~~~~~
+
+* Add `runtime_binary_dependencies` field to `python_tests` target (#10756)
+  `PR #10756 <https://github.com/pantsbuild/pants/pull/10756>`_
+  `PR #10744 <https://github.com/pantsbuild/pants/pull/10744>`_
+
+* Add new intrinsic `Paths` to avoid snapshotting (#10741)
+  `Issue #10711.
 <https://github.com/pantsbuild/pants/issues/10711.
>`_
```

Comment:
The ^M is bad. I'm not sure what that refers to. Generally, I delete the Issue part, and also related PRs. It's fine to only have the actual PR for the change in the links.

Reply:
Ditto on a couple of the changes in this. I'd recommend auditing that every entry only has a single link.

Reply:
`^M` would point at a line ending issue between Windows style (`\r\n`) and Unix style (`\n`)